packages/analyticsdx-template-lint/test/testutils.ts

```diff
@@ -13,6 +13,19 @@ import * as path from 'path';
 import * as readdirp from 'readdirp';
 import { promisify } from 'util';
 
+/** Set an object field based on a '.'-seperated name. */
+export function set(o: Record<string, unknown>, names: string, value: unknown) {
```

Comment:
This is kind of like a simple [lodash.set](https://lodash.com/docs/4.17.15#set), but I didn't want to bring in another library and we've avoided bringing in lodash so far.